tokio/src/runtime/metrics/worker.rs

```diff
@@ -1,8 +1,8 @@
-use crate::loom::sync::atomic::AtomicUsize;
-use crate::loom::sync::atomic::Ordering::Relaxed;
 use crate::runtime::metrics::Histogram;
 use crate::runtime::Config;
 use crate::util::metric_atomics::MetricAtomicU64;
+use std::sync::atomic::AtomicUsize;
+use std::sync::atomic::Ordering::Relaxed;
```

Comment:
Please add a comment about this not being the loom atomic.

Reply:
done — for clarity, could do a followup to create `MetricAtomicUsize` if we want

Reply:
That would make sense. But yes, let's do it in a follow-up.